x-pack/plugins/security_solution/cypress/integration/exceptions/add_edit_data_view_exception.spec.ts

```diff
@@ -0,0 +1,161 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { getException } from '../../objects/exception';
+import { getNewRule } from '../../objects/rule';
+
+import { ALERTS_COUNT, EMPTY_ALERT_TABLE, NUMBER_OF_ALERTS } from '../../screens/alerts';
+
+import { addExceptionFromFirstAlert, goToClosedAlerts, goToOpenedAlerts } from '../../tasks/alerts';
+import { createCustomRuleEnabled } from '../../tasks/api_calls/rules';
+import { goToRuleDetails } from '../../tasks/alerts_detection_rules';
+import { waitForAlertsToPopulate } from '../../tasks/create_new_rule';
+import { esArchiverLoad, esArchiverUnload, esArchiverResetKibana } from '../../tasks/es_archiver';
+import { login, visitWithoutDateRange } from '../../tasks/login';
+import {
+  addsException,
+  addsExceptionFromRuleSettings,
+  editException,
+  goToAlertsTab,
+  goToExceptionsTab,
+  removeException,
+  waitForTheRuleToBeExecuted,
+} from '../../tasks/rule_details';
+
+import { DETECTIONS_RULE_MANAGEMENT_URL } from '../../urls/navigation';
+import { deleteAlertsAndRules, postDataView } from '../../tasks/common';
+import {
+  EXCEPTION_EDIT_FLYOUT_SAVE_BTN,
+  EXCEPTION_ITEM_CONTAINER,
+  FIELD_INPUT,
+} from '../../screens/exceptions';
+import {
+  addExceptionEntryFieldValueOfItemX,
+  addExceptionEntryFieldValueValue,
+} from '../../tasks/exceptions';
+
+describe('Adds rule exception using data views', () => {
+  const NUMBER_OF_AUDITBEAT_EXCEPTIONS_ALERTS = '1 alert';
+
+  before(() => {
+    esArchiverResetKibana();
+    esArchiverLoad('exceptions');
+    login();
+
+    postDataView('exceptions-*').then((response) => {
+      cy.wrap({ dataViewId: response.body.index_pattern.id }).as('postDataView');
```

Comment:
<img width="787" alt="Screen Shot 2022-08-15 at 2 26 46 PM" src="https://user-images.githubusercontent.com/10927944/184722906-7ffc90e6-d28d-4d2a-a1c3-fe70c3c5e8a6.png">

I've got that value, but I'm not able to then get it using `this.postDataView.dataViewId`